.eslintrc.js

```diff
@@ -1,12 +1,4 @@
 module.exports = {
-    extends: ['google', 'plugin:react/recommended'],
```

Comment:
we should follow a coding style here no?

Reply:
So, I don't know why, I kept running into an error with the pre-processor once I added Storybook. My assumption is that the linter rules are in conflict since Storybook comes with one of its own?

Reply:
Ignore previous comment. I just found the issue and fixed it.